recipes/r-gsw/meta.yaml

```diff
@@ -0,0 +1,39 @@
+{% set version = "1.0-3" %}
+
+package:
+    name: r-gsw
+    version: {{ version | replace('-', '_') }}
+
+source:
+    fn: gsw_1.0-3.tar.gz
+    url:
+        - http://cran.r-project.org/src/contrib/gsw_{{ version }}.tar.gz
+        - http://cran.r-project.org/src/contrib/Archive/gsw/gsw_{{ version }}.tar.gz
```

Comment:
The double URL comes from the `conda skeleton cran <package>` boilerplate. I am not sure we need that.